src/styles/mixins.scss

```diff
@@ -0,0 +1,31 @@
+@mixin animation-minute-hand() {
+  @keyframes rotates {
+    from {
+      transform: rotate(0deg);
+    }
+    to {
+      transform: rotate(360deg);
+    }
+  }
```

Comment:
You have defined the same `@keyframes` animation rule twice in your mixins. This is not optimal and can lead to confusion. Instead, you can define it once outside of the mixins and reuse it in both. This way, if you need to change the animation in the future, you only need to do it in one place.